packages/manager/modules/pci/src/projects/project/kubernetes/details/service/service.html

```diff
@@ -229,45 +263,64 @@
                     </dd>
                 </dl>
             </div>
+
+            <!--OIDC-->
             <oui-tile-definition
-                data-term="{{:: 'kube_service_cluster_api_url' | translate }}"
+                data-term="{{:: 'kube_service_access_security_oidc_title' | translate }}"
             >
                 <oui-tile-description>
-                    <oui-clipboard
-                        data-model="$ctrl.cluster.url"
-                    ></oui-clipboard>
+
+                    <!--No OIDC provider-->
+                    <span
+                        data-ng-if="!$ctrl.oidcProvider.isDefined()"
+                        data-translate="kube_service_access_security_oidc_no_provider"
+                    ></span>
+
+                    <!--OIDC Provider-->
+                    <div data-ng-if="$ctrl.oidcProvider.isDefined()">
+                        <oui-clipboard
+                            data-model="$ctrl.oidcProvider.issuerUrl"
+                        ></oui-clipboard>
+
+                        <span data-ng-bind=":: $ctrl.oidcProvider.clientId"></span>
+                    </div>
+
                 </oui-tile-description>
+                <oui-action-menu aria-label="Actions" compact>
```

Comment:
`aria-label` not using translation ?

Reply:
Mistake from my side, updated.

Thanks